splice_machine/src/main/java/com/splicemachine/derby/stream/function/ScrollInsensitiveFunction.java

```diff
@@ -41,6 +41,9 @@ public ExecRow call(ExecRow execRow) throws Exception {
             this.operationContext.recordRead();
             op.setCurrentRow(execRow);
             this.operationContext.recordProduced();
+            if(!op.isForUpdate()) {
```

Comment:
@dgomezferro this prevents serializing rowId. Should we added to other functions that could be pushed down to Spark executors and used by the `ResultStreamer`?

Reply:
I think this is the only case, at least for now

Reply:
ok